presto-main/src/main/java/com/facebook/presto/sql/planner/plan/JoinNode.java

```diff
@@ -162,10 +162,7 @@ public PlanNode getRight()
     @JsonProperty("outputSymbols")
     public List<Symbol> getOutputSymbols()
     {
-        return ImmutableList.<Symbol>builder()
-                .addAll(left.getOutputSymbols())
-                .addAll(right.getOutputSymbols())
-                .build();
+        return outputSymbols;
```

Comment:
neh, now you can pull first commit without the latter. If you move it to first commit, tests will fail